packages/web-components/src/components/ic-breadcrumb-group/ic-breadcrumb-group.tsx

```diff
@@ -156,9 +171,7 @@ export class BreadcrumbGroup {
     collapsedBreadcrumbEl.id = "collapsed-ellipsis";
     collapsedBreadcrumbEl.innerText = "...";
     collapsedBreadcrumbEl.classList.add("collapsed-breadcrumb");
-    collapsedBreadcrumbEl.addEventListener("click", () => {
-      this.handleHiddenCollapsedBreadcrumbs(this.collapsedBreadcrumbWrapper);
-    });
+    collapsedBreadcrumbEl[method]("click", this.clickHandler);
```

Comment:
does this need to be applied in this way - can we just use addEventListener instead?